sharkfuser/templates/fusilliConfig.cmake.in

```diff
@@ -0,0 +1,9 @@
+@PACKAGE_INIT@
```

Comment:
Can we put this file under `build_tools/cmake/`? Let's keep the `templates` dir just for c++ interfaces.

Reply:
👍 . JFYI I thought the lowercase was load bearing but looks like I was wrong. CMake config file names can be `packagename-config.cmake` (lowercase is load bearing) or `PakageNameConfig.cmake` (lowercase not load bearing).